application/actions/auth.go

```diff
@@ -311,46 +323,44 @@ func AuthCallback(c buffalo.Context) error {
 	return c.Redirect(302, getLoginSuccessRedirectURL(authUser, returnTo))
 }
 
-// returnAuthError takes a error code and message and renders AuthResponse to json and returns
-func authError(c buffalo.Context, status int, code, message string) error {
-	resp := AuthResponse{
-		Error: &AuthError{
-			Code:    code,
-			Message: message,
-		},
+func authError(c buffalo.Context, code, message string, extras map[string]interface{}) error {
```

Comment:
Is it possible/helpful to define `extras` as a variadic so `emptyExtras` isn't needed?

Reply:
Good point.

Reply:
Ha, ha.  Making the change showed me I had a bug in authError ... I was merging `extras` into `allExtras`, but then not using `allExtras` anymore.